src/index.js

```diff
@@ -59,7 +59,7 @@ async function run(request, context) {
 the record is added, you can verify that it is set up correctly by making a POST request to this URL including
 the domain and domainkey parameters. For example:
 
-curl -X POST -F "domain=${domain}" -F "domainkey=${newkey}" ${currentURL}
+curl -X POST "${currentURL}?domain=${domain}&domainkey=${newkey}"
```

Comment:
```suggestion
curl -X POST -F "domain=${domain}" -F "domainkey=${newkey}" ${currentURL}
```
same here, this should remain a POST request